content/docs/deployment/csmoperator/troubleshooting/_index.md

```diff
@@ -57,4 +58,23 @@ Typical reasons for errors:
 * Incorrect driver type
 * Incorrect driver Spec env, args for containers
 * Incorrect RBAC permissions
-	
+
+### My CSM Replication install fails to validate replication prechecks with 'no such host'.
+In replication environments that utilize more than one cluster, and utilize FQDNs to reference API endpoints, it is highly recommended that the DNS be configured to resolve requests involving the FQDN to the appropriate Kubernetes cluster.
```

Comment:
I see it is mentioned as Kubernetes cluster. Isn't this issue only with OpenShift clusters. Should we also mention OpenShift at other places?